lib/spack/docs/developer_guide.rst

```diff
@@ -322,73 +322,78 @@ Stage objects
 Writing analyzers
 -----------------
 
-To write an analyzer, you can either include it in one of the files in the
-analyzers module folder (e.g., if it's associated with an application
-binary interface, ABI, you might add it to ``abi.py``) or you can add a new file.
-Your analyzer should be a subclass of the :class:`AnalyzerBase <spack.analyzers.AnalyzerBase>`,
-and you can look at other analyzers in that folder for examples. The guide
-here will tell you about the basic functions needed.
+To write an analyzer, you should add a new module file to the
```

Comment:
I think we should say "python module" or "python file" since we have sections all about entirely different "module files"